internal/distroregistry/distroregistry.go

```diff
@@ -26,6 +40,23 @@ func New(distros ...distro.Distro) (*Registry, error) {
 	return reg, nil
 }
 
+// NewDefault creates a Registry with all distributions supported by
+// osbuild-composer. If you need to add a distribution here, see the
+// supportedDistros variable.
+func NewDefault() *Registry {
+	var distros []distro.Distro
+	for _, distroInitializer := range supportedDistros {
+		distros = append(distros, distroInitializer())
+	}
+
+	registry, err := New(distros...)
+	if err != nil {
+		panic(fmt.Sprintf("two supported distros have the same name, this is a programming error: %v", err))
```

Comment:
Not super important: I think this error message would be better in the `New()` function, or maybe it's not necessary to have it in the first place? The line `return nil, fmt.Errorf("New: passed two distros with the same name: %s", d.Name())` says mostly the same.

Reply:
Yeah, good point, I will try to remember to change it in a follow-up. :)